server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -165,8 +165,10 @@ private void validateACLs(ServerContext context) {
 
```

Comment:
This does not need the stat.  Could use:
```
          final List<ACL> acls = zk.getACL(path, null);
```

Reply:
Do we have a minimum ZooKeeper version that is supported for Accumulo 2.1.0. Stat could not be null before ZooKeeper 3.5.0 (https://issues.apache.org/jira/browse/ZOOKEEPER-1222)